### DISCUSSION TITLE
HowTo build DELETE Request against Cloud REST API

### DISCUSSION THREAD

**Discussion by pwsimon** [dexie/Dexie.js#1845] [Q&A] [answered]
i'am using a [Compound Index](https://dexie.org/docs/Compound-Index) as PrimaryKey.
now i like to [DELETE](https://dexie.org/cloud/docs/rest-api#post-and-delete) entries using the REST API.

but HowTo build/encode the PrimaryKey with the Url?
HowTo sepearate the arguments? what about special chars e.g. square bracket?

i found the following Hint (but i have no clue how the JSON representation may look like)
> If primary key is [compound](https://dexie.org/docs/Compound-Index#compound-primary-key), the JSON representation of the key shall be given.

```
DELETE /public/<table>/<primaryKey> HTTP/1.1
Host: xxxx.dexie.cloud
Authorization: Bearer <token from /token endpoint (with GLOBAL_WRITE scope)>
Content-Type: application/json
```

**Comment by dfahlander** [comment, marked as answer]
I opened an issue #1846 believing this actually was an issue but found this was not the case. The docs have been updated in regards to this. See #1846.

**Reply by pwsimon** [reply]
it works. many thanks for the quick response 👏
BTW: the JSON representation looks a bit strange to me
see: [jsonurl](https://jsonurl.org/)
